CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+vNext (TBD)
+------------------
+
+The sync protocol version has been bumped to version 25. The server is backwards-compatible with clients using protocol version 24 or below, but clients at version 25 are not backwards-compatible with a server at protocol version 24. The server must be upgraded before any clients are upgraded.
+
+### Enhancements
+* Download progress is now reported to the server, even when there are no local changes. This allows the server to do history compaction much more aggressively, especially when there are many clients that rarely or never make local changes. ([#1772](https://github.com/realm/realm-dotnet/pull/1772))
+
+### Fixed
```

Comment:
I'm sure the sync upgrade fixes many issues - it might be a bit of work to enumerate those. But at least it should be mentioned that very critical bugs have been fixed.

Reply:
I believe it's all server-side fixes, do we expect any of these to affect clients?

Reply:
Yeah there was definitely some fixes that meant that old clients could introduce bugs (was that no-merge invariant)?